From 862d5215729db81da5ef240e1489533fe6c40a7d Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Thu, 11 Jan 2024 13:37:09 +0100
Subject: [PATCH 16/37] ia64: rename MAX_ORDER to MAX_PAGE_ORDER

MAX_ORDER was renamed to MAX_PAGE_ORDER in commit 5e0a760b44417
("mm, treewide: rename MAX_ORDER to MAX_PAGE_ORDER"). Rename it also
in the uses in ia64-specific code.

Fixes: 5e0a760b44417 ("mm, treewide: rename MAX_ORDER to MAX_PAGE_ORDER")
---
 arch/ia64/mm/hugetlbpage.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/ia64/mm/hugetlbpage.c b/arch/ia64/mm/hugetlbpage.c
index adc49f2d22e8..38de155d80c5 100644
--- a/arch/ia64/mm/hugetlbpage.c
+++ b/arch/ia64/mm/hugetlbpage.c
@@ -170,7 +170,7 @@ static int __init hugetlb_setup_sz(char *str)
 	size = memparse(str, &str);
 	if (*str || !is_power_of_2(size) || !(tr_pages & size) ||
 		size <= PAGE_SIZE ||
-		size > (1UL << PAGE_SHIFT << MAX_ORDER)) {
+		size > (1UL << PAGE_SHIFT << MAX_PAGE_ORDER)) {
 		printk(KERN_WARNING "Invalid huge page size specified\n");
 		return 1;
 	}
-- 
2.25.1

